fix the following runtime error :
    luajit: error while loading shared libraries: libluajit-5.1.so.2: cannot open shared object file: No such file or directory

by creating two symbolic links:
    libluajit-5.1.so -> libluajit-5.1.so.2.0.3
    libluajit-5.1.so.2 -> libluajit-5.1.so.2.0.3

Signed-off-by: Francois Perrad <francois.perrad@gadz.org>

Index: b/Makefile
===================================================================
--- a/Makefile
+++ b/Makefile
@@ -55,7 +55,7 @@
 INSTALL_STATIC= $(INSTALL_LIB)/$(INSTALL_ANAME)
 INSTALL_DYN= $(INSTALL_LIB)/$(INSTALL_SONAME)
 INSTALL_SHORT1= $(INSTALL_LIB)/$(INSTALL_SOSHORT)
-INSTALL_SHORT2= $(INSTALL_LIB)/$(INSTALL_SOSHORT)
+INSTALL_SHORT2= $(INSTALL_LIB)/$(INSTALL_SOSHORT).$(MAJVER)
 INSTALL_T= $(INSTALL_BIN)/$(INSTALL_TNAME)
 INSTALL_PC= $(INSTALL_PKGCONFIG)/$(INSTALL_PCNAME)
 
